mlflow_kubernetes_plugins/auth/middleware.py

```
@@ -232,6 +318,38 @@ def _k8s_auth_after_request(response: Response):
+                        status_code=response.status_code,
+                    )
+                if (
+                    auth_request is not None
+                    and rules is not None
+                    and identity is not None
+                    and isinstance(auth_request.workspace, str)
+                    and (workspace_name := auth_request.workspace.strip())
+                ):
+                    try:
+                        payload = json.loads(response.get_data(as_text=True))
+                    except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)